deploy/kustomize/image_registry/catalog_source.yaml

```diff
@@ -3,7 +3,6 @@ kind: CatalogSource
 metadata:
   name: kubevirt-hyperconverged
   namespace: openshift-marketplace
-  imagePullPolicy: Always
```

Comment:
can it be moved under spec? (in terms of passing validation)

Reply:
No, we get a similar failure.  I don't think imagePullPolicy supported at all in CatalogSource

Reply:
See also https://github.com/operator-framework/operator-lifecycle-manager/issues/903